core/lib/dal/src/consensus_dal/mod.rs

```diff
@@ -393,11 +406,11 @@ impl ConsensusDal<'_, '_> {
     pub async fn block_certificate(
         &mut self,
         block_number: validator::BlockNumber,
-    ) -> anyhow::Result<Option<validator::CommitQC>> {
+    ) -> anyhow::Result<Option<BlockCertificate>> {
         let Some(row) = sqlx::query!(
             r#"
             SELECT
-                certificate
+                versioned_certificate
```

Comment:
is it ok to only use `versioned_certificate` here, if so maybe it's worth a comment?

Reply:
No, it's not, I missed that one. I'll fix it.